### PR TITLE
Introduce more configuration for rails i18n

### PR DESCRIPTION
**Pull request by choubacha**
* Documents ignoring exceptions
* Allows the Regexp to be specified by config
* Can choose between 3 different sentence structures to evaluate

### CODE REVIEW

File: lib/rubocop/cop/i18n/rails_i18n/decorate_string.rb [hunk 1]
```diff
@@ -21,8 +21,59 @@ module RailsI18n
...
+        #   # bad
+        #
+        #   "Result is bad."
+        #
+        #   # good
+        #
+        #   t("result_is_good")
+        #   I18n.t("result_is_good")
+        #
+        # @example EnforcedSentenceType: fragemented_sentence
```
Comment (joeellis) [inline lib/rubocop/cop/i18n/rails_i18n/decorate_string.rb]: typo: `fragemented_sentence` -> `fragmented_sentence`

File: lib/rubocop/cop/i18n/rails_i18n/decorate_string.rb [hunk 2]
```diff
@@ -21,8 +21,59 @@ module RailsI18n
...
+        #
+        #   "Result is bad"   # Contains a capital to start
+        #   "result is bad."  # Ends in punctuation
+        #
+        #   # good
+        #
+        #   t("result_is_good")
+        #   I18n.t("result_is_good")
+        #
+        # @example EnforcedSentenceType: fragement
```
Comment (joeellis) [inline lib/rubocop/cop/i18n/rails_i18n/decorate_string.rb]: typo: `fragement` -> `fragment`

**Review by joeellis — APPROVED** [review]
Looking great from here!

File: lib/rubocop/cop/i18n/rails_i18n/decorate_string.rb (hunk 1 above)
Comment (choubacha) [reply lib/rubocop/cop/i18n/rails_i18n/decorate_string.rb]: I think I like my new spelling more 😜

**Review by joeltaylor — APPROVED** [review]
Awesome!

**Pull request merged**